crowdin_api/api_resources/translations/resource.py

```diff
@@ -325,26 +326,53 @@ def upload_translation(
         importEqSuggestions: Optional[bool] = None,
         autoApproveImported: Optional[bool] = None,
         translateHidden: Optional[bool] = None,
-    ):
+        addToTm: Optional[bool] = None,
+    ) -> dict:
         """
         Upload Translations.
 
+        Parameters
+        ----------
+        languageId: str
+            Language ID.
+        storageId: int
+            Storage ID.
+        fileId: int
+            File ID for import.
+        projectId: Optional[int]
+            Project ID.
+        importEqSuggestions: Optional[bool]
+            Define whether to add equal translations.
+        autoApproveImported: Optional[bool]
+            Mark uploaded translations as approved.
+        translateHidden: Optional[bool]
+            Allow translations upload to hidden source strings.
+        addToTm: Optional[bool]
+            Define whether to add translation to TM. Default: true
+
+        Returns
+        -------
+        dict
+            Upload Translation response from the API
```

Comment:
This type of parameter comment is not common in the project. I'd suggest removing it.

Reply:
Done!